doc/classes/@GlobalScope.xml

```diff
@@ -1110,6 +1118,17 @@
 				Creates a RID from a [param base]. This is used mainly from native extensions to build servers.
 			</description>
 		</method>
+		<method name="rotate_toward">
+			<return type="float" />
+			<param index="0" name="from" type="float" />
+			<param index="1" name="to" type="float" />
+			<param index="2" name="delta" type="float" />
+			<description>
+				Rotates [param from] toward [param to] by the [param delta] amount. Will not go past [param to].
+				Similar to [method move_toward], but interpolates correctly when the angles wrap around [constant @GDScript.TAU].
+				Use a negative [param delta] value to rotate toward the opposite angle.
```

Comment:
The behavior with negative delta makes sense but the docs can be clarified a bit:

```suggestion
				If [param delta] is negative, this function will rotate away from [param to], toward the opposite angle, and will not go past the opposite angle.
```